doc/server/vertx.md

```diff
@@ -14,7 +14,7 @@ to use this interpreter with `Future`.
 
 Then import the object:
 ```scala mdoc:compile-only
-import sttp.tapir.server.vertx.VertxFutureServerInterpreter._
+import sttp.tapir.server.vertx.interpreters.VertxFutureServerInterpreter._
```

Comment:
I think I'd keep the interpreters in the "root" package, that is `sttp.tapir.server.vertx`. Easier for users :)